editor/modes/visual-editor/block.js

```diff
@@ -104,6 +147,10 @@ class VisualEditorBlock extends wp.element.Component {
 		}
 	}
 
+	stopTyping() {
```

Comment:
You mean binding `this.props.onStopTying` as the event handler? It's a little concerning to me since the `onStopTyping` prop may not be the same in the constructor as it is when the component later unmounts (or the event is bound), so we risk zombie event listeners.